extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -688,11 +689,13 @@ export class AmpStoryPage extends AMP.BaseElement {
             break;
           case 'amp-audio':
           case 'amp-video':
-            if (mediaEl.readyState >= 2) {
+            if (
+              mediaEl.readyState == ReadyState.LOADING ||
+              mediaEl.readyState == ReadyState.COMPLETE
```

Comment:
`readyState` for mediaEls are not the same as the HTML readyStates (which are numbers), they are `ReadyState` enum values.